config/settings.yml

```diff
@@ -97,6 +97,7 @@ attachments_dir: ":rails_root/public/attachments"
 basic_auth_support: true
 # Enable legacy OAuth 1.0 support
 oauth_10_support: true
+oauth_10_registration: true
```

Comment:
I think just a simple test for the `:new -> :index` redirect would be needed and is also worthwhile.

Reply:
I've added a test and then spent ages wondering why it broke everything only to eventually figure out that yesterday's bundle update gave us a buggy bootsnap!

Reply:
> yesterday's bundle update gave us a buggy bootsnap!

Sorry I found that a couple of hours ago, but I was still deep-diving on something else so hadn't made a PR for bootsnap yet!